packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -565,6 +565,14 @@ export default class TradeStore extends BaseStore {
                                 type: response.msg_type,
                                 ...response.error,
                             });
+
+                            // Clear purchase info on mobile after toast box error disappears (mobile_toast_timeout = 3500)
+                            if (isMobile() && this.root_store.common?.services_error?.type === 'buy') {
+                                setTimeout(() => {
+                                    this.clearPurchaseInfo();
+                                    this.requestProposal();
```

Comment:
can we only clearpurchase info here as per the comment, and resubscribe propsal should be done right away after forget proposal

btw, any idea why do we unsubscribe/forget proposal once login toastbox is showing?

Reply:
@mustofa-binary we need to invoke `requestProposal` here to reinitialise proposals and reset purchase states otherwise we the issue persists.